web/frontend/src/pages/ballot/components/Select.tsx

```diff
@@ -93,22 +100,25 @@ const Select: FC<SelectProps> = ({ select, answers, setAnswers, language }) => {
       <div className="sm:pl-8 mt-2 pl-6">
         {Array.from(answers.SelectAnswers.get(select.ID).entries())
           .map(([choiceIndex, isChecked]) => {
-            if (language === 'fr' && select.ChoicesMap.has('fr'))
+            if (language === 'fr' && select.ChoicesMap.ChoicesMap.has('fr'))
```

Comment:
```suggestion
            if (select.ChoicesMap.ChoicesMap.has(language))
```

Same as above, then you only have 2 ifs and it works even if there should be more languages.